libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -170,6 +170,21 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS", 17, AC_PrecLand, _options, 0),
 
+    // @Param: ORIENT
+    // @DisplayName: Camera Orientation
+    // @Description: Orientation of camera/sensor on body
+    // @Values: 0:Forward, 4:Back, 25:Down
+    // @User: Advanced
+    AP_GROUPINFO("ORIENT", 18, AC_PrecLand, _orient, ROTATION_PITCH_270),
+
+    // @Param: CAM_FOV
+    // @DisplayName: Camera Field of View
+    // @Description: Horizontal field of view of camera 
```

Comment:
This came up while implementing a different heading correction algorithm (we dropped the idea of using that algorithm though) but this changes looked good to me as it used [here](https://github.com/shiv-tyagi/ardupilot/blob/0d6145111bc59cddab747edbb940d82fddfe6688/libraries/AC_PrecLand/AC_PrecLand.cpp#L637) to reject target vectors which are not in camera fov. Earlier we were rejecting all the vectors which are behind the vehicle (that's where the idea of 180.0f fov default came). With this we can do that pruning at a finer level.

Reply:
How could we have readings that are outside the camera's field of view though?  Surely physically the camera cannot report distances that are outside its field of view.  I also think we should remove this parameter if possible because users will fuss over it trying to get the exact FOV which increases the setup effort and (I think) it's not really required.

Reply:
I agree! I'll write a PR to set simulated sensor's FOV. Will remove this from here.